doc/src/build/programming-with-objects/ch5-child-objects.md

```diff
@@ -55,57 +53,59 @@ Similarly, we can define an API to create an object of `Parent` type:
 ```rust
 public entry fun create_parent(ctx: &mut TxContext) {
     let parent = Parent {
-        info: object::new(ctx),
+        id: object::new(ctx),
         child: option::none(),
     };
     transfer::transfer(parent, tx_context::sender(ctx));
 }
 ```
-Since the `child` field is `Option` type, we can start with `Option::none()`.
+Since the `child` field is `Option` type, we can start with `option::none()`.
 Now we can define an API that makes an object of `Child` a child of an object of `Parent`:
 ```rust
 public entry fun add_child(parent: &mut Parent, child: Child) {
-    let child_ref = transfer::transfer_to_object(child, parent);
-    option::fill(&mut parent.child, child_ref);
+    option::fill(&mut parent.child, object::id(&child));
+    transfer::transfer_to_object(child, parent);
 }
 ```
-This function takes `child` by value, calls `transfer_to_object` to transfer the `child` object to the `parent`, and returns a `child_ref`.
-After that, we can fill the `child` field of `parent` with `child_ref`.
-If we comment out the second line, the Move compiler will complain that we cannot drop `child_ref`.
+This function takes `child` by value, fills the `child` field of `parent` with the `ID` of the `Child` object, and calls `transfer_to_object` to transfer the `child` object to the `parent`.
```

Comment:
fills the `child` field of `parent` with the `ID` of the `Child` object, and calls `transfer_to_object` to transfer the `child` object to the `parent`

If it passes the `ID` of the `Child` object, does it transfer the `Child` object to the `Parent` ?
Step 2 following this says that after the call the Parent object owns a Child object

Sorry if this is just me getting lost with all the childs mentioned